internal/backends/winit/winitwindowadapter.rs

```diff
@@ -478,16 +664,40 @@ impl WindowAdapter for WinitWindowAdapter {
     }
 
     fn position(&self) -> Option<corelib::api::PhysicalPosition> {
-        match self.winit_window().outer_position() {
-            Ok(outer_position) => {
-                Some(corelib::api::PhysicalPosition::new(outer_position.x, outer_position.y))
+        match &*self.winit_window_or_none.borrow() {
+            WinitWindowOrNone::HasWindow(winit_window) => match winit_window.outer_position() {
+                Ok(outer_position) => {
+                    Some(corelib::api::PhysicalPosition::new(outer_position.x, outer_position.y))
+                }
+                Err(_) => None,
+            },
+            WinitWindowOrNone::None(attributes) => {
+                attributes.borrow().position.and_then(|pos| {
+                    match pos {
+                        winit::dpi::Position::Physical(phys_pos) => {
+                            Some(corelib::api::PhysicalPosition::new(phys_pos.x, phys_pos.y))
```

Comment:
I guess that's because these are the only places where we convert the positions, while we have physical_size_to_slint/winit because we convert sizes in a few more places.